lib/recognizer_web/router.ex

```diff
@@ -96,7 +97,6 @@ defmodule RecognizerWeb.Router do
 
     get "/settings", UserSettingsController, :edit
     put "/settings", UserSettingsController, :update
-    get "/settings/two-factor", UserSettingsController, :two_factor
-    post "/settings/two-factor", UserSettingsController, :two_factor_confirm
+    live "/settings/two-factor", TwoFactorSettingsLive
```

Comment:
heads up, you could specify the `:index` action here, which gives you `Routes.two_factor_settings_path(@conn, :index)`. I found this more consistent with other path helpers instead of all liveview paths going through `live_path`. 

it's a little awkward though because there's no real `:index` action; we're just informing how to create the route.